api/src/main/java/org/openmrs/module/fhir2/api/dao/impl/FhirObservationDaoImpl.java

```diff
@@ -37,22 +37,14 @@
 @Component
 public class FhirObservationDaoImpl extends BaseDaoImpl implements FhirObservationDao {
 	
-	@Autowired
-	@Qualifier("sessionFactory")
-	private SessionFactory sessionFactory;
-	
-	@Override
-	public Obs getObsByUuid(String uuid) {
-		return (Obs) sessionFactory.getCurrentSession().createCriteria(Obs.class).add(eq("uuid", uuid)).uniqueResult();
-	}
 	
 	@Override
 	public Collection<Obs> searchForObservations(ReferenceAndListParam encounterReference,
 	        ReferenceAndListParam patientReference, ReferenceParam hasMemberReference, TokenAndListParam valueConcept,
 	        DateRangeParam valueDateParam, QuantityAndListParam valueQuantityParam, StringAndListParam valueStringParam,
 	        DateRangeParam date, TokenAndListParam code, SortSpec sort) {
 		
-		Criteria criteria = sessionFactory.getCurrentSession().createCriteria(Obs.class);
+		Criteria criteria = getsessionFactory.getCurrentSession().createCriteria(Obs.class);
```

Comment:
```java
Criteria criteria = getSessionFactory().getCurrentSession().createCriteria(Obs.class);
```

Reply:
This line of code is ok , though its also complaining of defining the sessionFactory

Reply:
Because it's already defined in `BaseFhirDaoImpl`

Reply:
Finished

Reply:
@sherrif10 Can you push that change out to this branch so it gets updated here? Thanks!

Reply:
@ibacher yes let me do that now

Reply:
This implementation was changed so i wont extend to BaseFhirDaoImpl since you made changes